reflections.md

```diff
@@ -0,0 +1,17 @@
+# Capstone Reflections
+With any large project that you've undertaken, especially one which requires a lot of personal growth and self-improvement, it's important to take time to reflect on the your process for completing the project and the outcomes of the decisions you made along the way.
+
+We're providing a set of suggested reflection prompts / questions that you can use to consider how things went with your capstone project, and we encourage you to add your own prompts as well!
+
+## Reflection prompts
+1. What went well during the capstone project?
+1. What didn't go as well?
+1. What aspects of the project were more difficult than anticipated?
+1. What aspects were less difficult than anticipated?
+1. What features in your project were not possible in the time frame provided? Were any of them critical to the success of the project?
+
+1. How did your project plan shift as the project progressed?
+1. If any project features took longer than anticipated to complete, how did you modify the rest of your planned features to accommodate having less time?
+1. How did your intended plan of action line up with the reality?
+
+1. Is there anything you would do differently for the next large project you undertake?
```

Comment:
I think these questions are good - I think we are missing some reflections on learning goals which I think is really the most important. I'd like the overall impact of the reflection questions to be less about the capstone project itself and more about what the takeaways are going to be towards internship and future projects.